### ISSUE TITLE
Feature Request: Dual lightgun emulation through raw input

### ISSUE THREAD

**Issue by gustavoalara** [libretro/swanstation#6]
Could be nice if we can play PS1 lightgun games with two players/lightguns as we can do on Flycast, Mame, and another cores, through raw input on Windows or uinput on *nix 

Thanks a lot

**Comment by shiryu55** [comment]
I totally think it would be great update

**Comment by Widge-5** [comment]
Agreed.  I use a RetroPie/EmulationStation on a Raspberry Pi.  lr-pcsx-rearmed is incapabable of running Judge Dredd, but lr-swanstation can. However, there's only support for a single lightgun.  Being able to use 2 lightguns in this game would be fantastic.
